src/megaapi_impl.cpp

```diff
@@ -21659,21 +21660,23 @@ void MegaApiImpl::sendPendingRequests()
                 syncConfig.setError(syncError);
             }
 
-            client->copySyncConfig(syncConfig, [this, request](UnifiedSync *unifiedSync, const SyncError &syncError, error e)
-            {
-                if (!e && !unifiedSync)
-                {
-                    e = API_ENOENT;
-                }
+            client->ensureSyncUserAttributes([this, request, syncConfig](Error e){
```

Comment:
No error management?? If `e` reports an error, we cannot continue with this request.

Reply:
Fixed in 4d41619